verification/curator-service/ui/src/components/App.tsx

```diff
@@ -54,6 +86,8 @@ function Home(): JSX.Element {
             <br />
             <Link to="/sources">Sources</Link>
             <br />
+            <Link to="/profile">Profile</Link>
```

Comment:
only render this one conditionally if there's a user in the state?

Reply:
Good idea. I removed the Link in Home and also removed the /profile Route so it can't be navigated to manually.